meinberlin/apps/projects/serializers.py

```diff
@@ -62,6 +63,7 @@ class ProjectSerializer(serializers.ModelSerializer, CommonFields):
     title = serializers.SerializerMethodField()
     type = serializers.SerializerMethodField()
     url = serializers.SerializerMethodField()
+    topics = serializers.SerializerMethodField()
```

Comment:
This was all just hacked together to test, I think having a`TopicsSerializer` which we add here instead of the function would probably be nice ?

Reply:
true